package.json

```diff
@@ -96,6 +96,7 @@
     "babel-plugin-typescript-to-proptypes": "1.4.2",
     "browserslist": "4.21.5",
     "bundlesize": "0.18.1",
+    "cac": "^6.7.14",
```

Comment:
I was wondering if we should use a fixed version as we do with the rest of the dependencies 🤔